services/slam/builtin/builtin.go

```diff
@@ -357,8 +357,8 @@ func configureCameras(ctx context.Context, svcConfig *AttrConfig, deps registry.
 				return "", nil, errors.Wrap(err, "error getting camera properties for slam service")
 			}
 
-			brownConrady, ok := props.DistortionParams.(*transform.BrownConrady)
-			if !ok || brownConrady == nil {
+			_, ok = props.DistortionParams.(*transform.BrownConrady)
+			if !ok {
 				return "", nil, errors.New("error getting distortion_parameters for slam service, only BrownConrady distortion parameters are supported")
```

Comment:
Please add back the test case for this error in slam/builtin_test.go

Reply:
Sorry, but unless I'm missing something, I don't think I removed any test cases as part of this change. What did I remove which you would like me to add back?